### DISCUSSION TITLE
Second Netbox

### DISCUSSION THREAD

**Discussion by DoktorByte** [netbox-community/netbox#6819] [Q&A] [answered]
Good morning all,

we have a productive Netbox running with version (v2.10.9).
I have now prepared a second VM with the same version and configuration.
With the empty default database I can now log in etc..
Now I have made a dump of the database on the productive Netbox, and imported this on the second VM.
Now I get the following error:

![grafik](https://user-images.githubusercontent.com/58031943/127105426-df434f87-6dc3-4bf9-bbaa-7805b659f6e0.png)

Does anyone have any idea where my error is?

Thank you and best regards

**Comment by yrro** [comment]
What happened to the `django_content_type` table--was it included in your database dump?

**Reply by DoktorByte** [reply]
Hello,

yes the table is included in the dump and the content is the same on both systems:

netbox=# select * from django_content_type;
 id |   app_label    |           model
----+----------------+---------------------------
  1 | dcim           | consoleport
  2 | dcim           | consoleserverport
  3 | dcim           | powerport
  4 | dcim           | poweroutlet
  5 | dcim           | interface
  7 | circuits       | circuittermination
  8 | admin          | logentry
  9 | auth           | permission
 10 | auth           | group
 11 | auth           | user
 12 | contenttypes   | contenttype
 13 | sessions       | session
 14 | taggit         | tag
 15 | taggit         | taggeditem
 16 | circuits       | circuittype
 17 | circuits       | provider
 18 | circuits       | circuit
 19 | dcim           | consoleporttemplate
 20 | dcim           | consoleserverporttemplate
 21 | dcim           | device
 22 | dcim           | devicerole
 23 | dcim           | devicetype
 24 | dcim           | interfacetemplate
 25 | dcim           | manufacturer
 26 | dcim           | platform
 27 | dcim           | poweroutlettemplate
 28 | dcim           | powerporttemplate
 29 | dcim           | rack
 30 | dcim           | rackgroup
 31 | dcim           | site
 32 | dcim           | devicebaytemplate
 33 | dcim           | devicebay
 34 | dcim           | rackrole
 35 | dcim           | rackreservation
 36 | dcim           | region
 37 | dcim           | inventoryitem
 38 | dcim           | virtualchassis
 39 | dcim           | rearport
 40 | dcim           | rearporttemplate
 41 | dcim           | frontporttemplate
 42 | dcim           | frontport
 43 | dcim           | cable
 44 | dcim           | powerfeed
 45 | dcim           | powerpanel
 46 | ipam           | aggregate
 47 | ipam           | ipaddress
 48 | ipam           | prefix
 49 | ipam           | rir
 50 | ipam           | role
 51 | ipam           | vlan
 52 | ipam           | vrf
 53 | ipam           | vlangroup
 54 | ipam           | service
 55 | extras         | customfield
 57 | extras         | exporttemplate
 60 | extras         | imageattachment
 62 | extras         | webhook
 63 | extras         | objectchange
 64 | extras         | configcontext
 65 | extras         | tag
 66 | extras         | taggeditem
 67 | extras         | customlink
 68 | extras         | script
 69 | secrets        | secretrole
 70 | secrets        | userkey
 71 | secrets        | sessionkey
 72 | secrets        | secret
 73 | tenancy        | tenantgroup
 74 | tenancy        | tenant
 75 | users          | token
 76 | virtualization | cluster
 77 | virtualization | clustergroup
 78 | virtualization | clustertype
 79 | virtualization | virtualmachine
 80 | users          | userconfig
 81 | virtualization | vminterface
 82 | extras         | report
 83 | dcim           | cablepath
 84 | ipam           | routetarget
 85 | extras         | jobresult
 86 | users          | admingroup
 87 | users          | adminuser
 88 | users          | objectpermission
(83 rows)

**Comment by candlerb** [comment, marked as answer]
It could be a permissions problem. What happens if you connect to the new database using the exact same credentials as Netbox itself uses (in `configuration.py`)?

**Reply by DoktorByte** [reply]
Hello,
yes the user had the appropriate permissions 'GRANT ALL PRIVILEGES ON DATABASE netbox TO netbox;' and could also log in.

However, I noticed during this test that with a \dt no schemas were displayed.

I have now dumped the database again on the productive side and imported it on the standby side and now everything works.

No idea why the schemas were missing

Thanks and best regards

**Reply by yrro** [reply]
Tables imported into the wrong schema? Try `\dn` and then for each schema listed, `\dt schemaname.` to list all the tables in the schema.
